Qa utils

roiskhoiron opened the pull request.

roiskhoiron committed:
fix: adjust to record and download video file

roiskhoiron committed:
fix: create demo program to implement sdk

roiskhoiron committed:
fix: implement SDK

roiskhoiron committed:
fix: startRecording() is performed after the correct permissions are obtained

roiskhoiron committed:
feat: Notification bar as controller record capture video

roiskhoiron committed:
feat: auto stop and record video as file event forced kill by user or crash app

roiskhoiron committed:
feat: stop record log and video as file

roiskhoiron committed:
feat: auto stop record log and video as file event emergency crash or forced kill by user

roiskhoiron committed:
feat: boilerplate code simplification

roiskhoiron committed:
fix: try record on 2nd occasion and so on

roiskhoiron committed:
fix: flow solved

roiskhoiron committed:
fix: all is well

roiskhoiron committed:
feat: create multi-environment with flavours gradle and implement into program code logic

roiskhoiron committed:
feat: add welcome QA screen on staging flavour

JGeraldoLima commented:
@roiskhoiron  hey man, are you sure this PR is for this repo (maybe you tried to fork it)? There's no description at all about what you are trying to do... can you give me more details about it?

roiskhoiron committed:
style: revamp skin of QA onboarding layout

roiskhoiron committed:
style: revamp skin of qa on boarding

roiskhoiron committed:
Setup QA Snap SDK for Maven publishing

- Add Maven publishing configuration for JitPack and Maven Central
- Add comprehensive SDK documentation (SDK_README.md)
- Add publishing guides and automation scripts
- Configure build.gradle for proper publishing
- Add jitpack.yml for JitPack compatibility
- Update .gitignore for publishing security
- Ready for v1.0.0 release

roiskhoiron committed:
Fix JitPack build: Update to Java 17 for AGP 8.3.1 compatibility
